cpp/arcticdb/version/local_versioned_engine.cpp

```diff
@@ -249,6 +249,42 @@ std::pair<VersionedItem, FrameAndDescriptor> LocalVersionedEngine::read_datafram
     return std::make_pair(version.value_or(VersionedItem{}), std::move(frame_and_descriptor));
 }
 
+std::pair<VersionedItem, py::object> LocalVersionedEngine::read_descriptor_version_internal(
```

Comment:
This method is new. What was the previous method that was used for this?

Reply:
Okay, plain `read_descriptor`

Reply:
I tried to mimic similar structure than read_batch/read, which also share the internal_... function